src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -17,12 +23,21 @@
    [quo2/icon :i/arrow-left {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
 
 (defn options-button []
-  [quo2/button {:type                :grey
-                :size                32
-                :width               32
-                :style               {:margin-right 20}
-                :accessibility-label :options-button}
-   [quo2/icon :i/options {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
+  (let [group (rf/sub [:chats/current-chat])]
+    [quo2/button {:type                :grey
+                  :size                32
+                  :width               32
+                  :style               {:margin-right 20}
+                  :accessibility-label :options-button
+                  :on-press            #(rf/dispatch [:bottom-sheet/show-sheet
+                                                      {:content (fn [] [actions/group-details-actions group])}])}
```

Comment:
FYI: Not a suggestion, just for the curious, you can use `constantly` here instead of the anonymous function. Some people like it.

Reply:
@ilmotta I am not sure what you mean?

Reply:
Hey @OmarBasem, `constantly` is a core Clojure function and some people like to use it to communicate that a function doesn't care about its arguments and the return value is always the same. It has its fair share of usages in other contexts too.

This is just FYI stuff, I don't even think it's better/worse.

```clojure
(constantly [actions/group-details-actions group])
```